Automattic/wp-calypso
# Pricing: Align the Feature Grid With Our Marketing Pages

We recently updated the contents of the Enterprise plan offering in the marketing pricing grid. This PR align the Calypso plans and pricing component, including the following chnages:

* Updated the set of client logos: pcNC1U-1ch-p2#comment-1476
* Introduced the list of Enterprise features: pcNC1U-1ch-p2
    * Unlike the other feature references coming from `@calypso/products`, the new Enterprise features are displayed only in the pricing grid and aren’t used for plan comparison. They don’t include any additional pieces of content like tooltips, either.

Unrelated to the Enterprise plan, but also included:

* Removed the references to `Referral and loyalty programs`: 3074-gh-Automattic/martech

---

All the changes mentioned above have already been implemented on our landing pages.

![Screenshot](https://github.com/user-attachments/assets/a49a0e0c-cc00-4006-aef1-6d7bda568bfa)

## Testing Instructions

* Navigate to `/start/plans`.
* Ensure the changes are visible and functional in both wide and narrow viewports.
    * The Enterprise features should look and behave as the other plans’ features.

## Pre-merge Checklist
